test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -345,6 +345,25 @@ TEST_F(ClusterManagerImplTest, SubsetLoadBalancerRestriction) {
       "cluster: cluster type 'original_dst' may not be used with lb_subset_config");
 }
 
+TEST_F(ClusterManagerImplTest, RingHashLoadBalancerInitialization) {
+  const std::string json = R"EOF(
```

Comment:
If possible can you do a v2 YAML test also? We have been adding new tests w/ v2 YAML config. See some recent PRs for examples on how to do it. It's pretty easy.

Reply:
Added one